{
  "uid" : "16ba88760c75605f",
  "name" : "Verify switching of the 'ru', 'en' buttons in the Header on the Start Unauthorized Page",
  "fullName" : "tests.header_test.TestUnauthHeaderPageLinks#test_hpu_03_06_verify_unauth_ru_en_buttons_switching",
  "historyId" : "9e0c459a9826c9d8cea91670de01fceb",
  "time" : {
    "start" : 1728694576821,
    "stop" : 1728694576821,
    "duration" : 0
  },
  "status" : "broken",
  "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55acbfcb602a <unknown>\n#1 0x55acbf99c5e0 <unknown>\n#2 0x55acbf9947eb <unknown>\n#3 0x55acbf984a09 <unknown>\n#4 0x55acbf986744 <unknown>\n#5 0x55acbf984cbd <unknown>\n#6 0x55acbf98452f <unknown>\n#7 0x55acbf984422 <unknown>\n#8 0x55acbf982493 <unknown>\n#9 0x55acbf982ada <unknown>\n#10 0x55acbf99ef41 <unknown>\n#11 0x55acbfa308c5 <unknown>\n#12 0x55acbfa10b22 <unknown>\n#13 0x55acbfa2fd7d <unknown>\n#14 0x55acbfa108c3 <unknown>\n#15 0x55acbf9de6b3 <unknown>\n#16 0x55acbf9df68e <unknown>\n#17 0x55acbfc80a2b <unknown>\n#18 0x55acbfc849b1 <unknown>\n#19 0x55acbfc6d225 <unknown>\n#20 0x55acbfc85532 <unknown>\n#21 0x55acbfc5238f <unknown>\n#22 0x55acbfca4f28 <unknown>\n#23 0x55acbfca50f3 <unknown>\n#24 0x55acbfcb4e7c <unknown>\n#25 0x7fe01d894ac3 <unknown>",
  "statusTrace" : "driver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"f82ff9a3a5d926057fa82c59f5b345f9\")>\n\n    @pytest.fixture()\n    @allure.step(f'Open page: {MainPageLinks.URL_MAIN_PAGE}')\n    def main_page_open(driver):\n>       driver.get(MainPageLinks.URL_MAIN_PAGE)\n\ntests/conftest.py:22: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:353: in get\n    self.execute(Command.GET, {\"url\": url})\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py:344: in execute\n    self.error_handler.check_response(response)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.remote.errorhandler.ErrorHandler object at 0x7f8f5b306ea0>\nresponse = {'status': 500, 'value': '{\"value\":{\"error\":\"unknown error\",\"message\":\"unknown error: net::ERR_CONNECTION_REFUSED\\\\n  ...\\\\n#23 0x55acbfca50f3 \\\\u003Cunknown>\\\\n#24 0x55acbfcb4e7c \\\\u003Cunknown>\\\\n#25 0x7fe01d894ac3 \\\\u003Cunknown>\\\\n\"}}'}\n\n    def check_response(self, response: Dict[str, Any]) -> None:\n        \"\"\"Checks that a JSON response from the WebDriver does not have an\n        error.\n    \n        :Args:\n         - response - The JSON response from the WebDriver server as a dictionary\n           object.\n    \n        :Raises: If the response contains an error message.\n        \"\"\"\n        status = response.get(\"status\", None)\n        if not status or status == ErrorCode.SUCCESS:\n            return\n        value = None\n        message = response.get(\"message\", \"\")\n        screen: str = response.get(\"screen\", \"\")\n        stacktrace = None\n        if isinstance(status, int):\n            value_json = response.get(\"value\", None)\n            if value_json and isinstance(value_json, str):\n                import json\n    \n                try:\n                    value = json.loads(value_json)\n                    if len(value) == 1:\n                        value = value[\"value\"]\n                    status = value.get(\"error\", None)\n                    if not status:\n                        status = value.get(\"status\", ErrorCode.UNKNOWN_ERROR)\n                        message = value.get(\"value\") or value.get(\"message\")\n                        if not isinstance(message, str):\n                            value = message\n                            message = message.get(\"message\")\n                    else:\n                        message = value.get(\"message\", None)\n                except ValueError:\n                    pass\n    \n        exception_class: Type[WebDriverException]\n        e = ErrorCode()\n        error_codes = [item for item in dir(e) if not item.startswith(\"__\")]\n        for error_code in error_codes:\n            error_info = getattr(ErrorCode, error_code)\n            if isinstance(error_info, list) and status in error_info:\n                exception_class = getattr(ExceptionMapping, error_code, WebDriverException)\n                break\n        else:\n            exception_class = WebDriverException\n    \n        if not value:\n            value = response[\"value\"]\n        if isinstance(value, str):\n            raise exception_class(value)\n        if message == \"\" and \"message\" in value:\n            message = value[\"message\"]\n    \n        screen = None  # type: ignore[assignment]\n        if \"screen\" in value:\n            screen = value[\"screen\"]\n    \n        stacktrace = None\n        st_value = value.get(\"stackTrace\") or value.get(\"stacktrace\")\n        if st_value:\n            if isinstance(st_value, str):\n                stacktrace = st_value.split(\"\\n\")\n            else:\n                stacktrace = []\n                try:\n                    for frame in st_value:\n                        line = frame.get(\"lineNumber\", \"\")\n                        file = frame.get(\"fileName\", \"<anonymous>\")\n                        if line:\n                            file = f\"{file}:{line}\"\n                        meth = frame.get(\"methodName\", \"<anonymous>\")\n                        if \"className\" in frame:\n                            meth = f\"{frame['className']}.{meth}\"\n                        msg = \"    at %s (%s)\"\n                        msg = msg % (meth, file)\n                        stacktrace.append(msg)\n                except TypeError:\n                    pass\n        if exception_class == UnexpectedAlertPresentException:\n            alert_text = None\n            if \"data\" in value:\n                alert_text = value[\"data\"].get(\"text\")\n            elif \"alert\" in value:\n                alert_text = value[\"alert\"].get(\"text\")\n            raise exception_class(message, screen, stacktrace, alert_text)  # type: ignore[call-arg]  # mypy is not smart enough here\n>       raise exception_class(message, screen, stacktrace)\nE       selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\nE         (Session info: chrome=129.0.6668.89)\nE       Stacktrace:\nE       #0 0x55acbfcb602a <unknown>\nE       #1 0x55acbf99c5e0 <unknown>\nE       #2 0x55acbf9947eb <unknown>\nE       #3 0x55acbf984a09 <unknown>\nE       #4 0x55acbf986744 <unknown>\nE       #5 0x55acbf984cbd <unknown>\nE       #6 0x55acbf98452f <unknown>\nE       #7 0x55acbf984422 <unknown>\nE       #8 0x55acbf982493 <unknown>\nE       #9 0x55acbf982ada <unknown>\nE       #10 0x55acbf99ef41 <unknown>\nE       #11 0x55acbfa308c5 <unknown>\nE       #12 0x55acbfa10b22 <unknown>\nE       #13 0x55acbfa2fd7d <unknown>\nE       #14 0x55acbfa108c3 <unknown>\nE       #15 0x55acbf9de6b3 <unknown>\nE       #16 0x55acbf9df68e <unknown>\nE       #17 0x55acbfc80a2b <unknown>\nE       #18 0x55acbfc849b1 <unknown>\nE       #19 0x55acbfc6d225 <unknown>\nE       #20 0x55acbfc85532 <unknown>\nE       #21 0x55acbfc5238f <unknown>\nE       #22 0x55acbfca4f28 <unknown>\nE       #23 0x55acbfca50f3 <unknown>\nE       #24 0x55acbfcb4e7c <unknown>\nE       #25 0x7fe01d894ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py:229: WebDriverException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "clear_allure_results_folder",
    "time" : {
      "start" : 1728694482412,
      "stop" : 1728694482412,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "main_page_open",
    "time" : {
      "start" : 1728694577588,
      "stop" : 1728694578220,
      "duration" : 632
    },
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55acbfcb602a <unknown>\n#1 0x55acbf99c5e0 <unknown>\n#2 0x55acbf9947eb <unknown>\n#3 0x55acbf984a09 <unknown>\n#4 0x55acbf986744 <unknown>\n#5 0x55acbf984cbd <unknown>\n#6 0x55acbf98452f <unknown>\n#7 0x55acbf984422 <unknown>\n#8 0x55acbf982493 <unknown>\n#9 0x55acbf982ada <unknown>\n#10 0x55acbf99ef41 <unknown>\n#11 0x55acbfa308c5 <unknown>\n#12 0x55acbfa10b22 <unknown>\n#13 0x55acbfa2fd7d <unknown>\n#14 0x55acbfa108c3 <unknown>\n#15 0x55acbf9de6b3 <unknown>\n#16 0x55acbf9df68e <unknown>\n#17 0x55acbfc80a2b <unknown>\n#18 0x55acbfc849b1 <unknown>\n#19 0x55acbfc6d225 <unknown>\n#20 0x55acbfc85532 <unknown>\n#21 0x55acbfc5238f <unknown>\n#22 0x55acbfca4f28 <unknown>\n#23 0x55acbfca50f3 <unknown>\n#24 0x55acbfcb4e7c <unknown>\n#25 0x7fe01d894ac3 <unknown>\n\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
    "steps" : [ {
      "name" : "Open page: https://brainup.site/",
      "time" : {
        "start" : 1728694577588,
        "stop" : 1728694578220,
        "duration" : 632
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55acbfcb602a <unknown>\n#1 0x55acbf99c5e0 <unknown>\n#2 0x55acbf9947eb <unknown>\n#3 0x55acbf984a09 <unknown>\n#4 0x55acbf986744 <unknown>\n#5 0x55acbf984cbd <unknown>\n#6 0x55acbf98452f <unknown>\n#7 0x55acbf984422 <unknown>\n#8 0x55acbf982493 <unknown>\n#9 0x55acbf982ada <unknown>\n#10 0x55acbf99ef41 <unknown>\n#11 0x55acbfa308c5 <unknown>\n#12 0x55acbfa10b22 <unknown>\n#13 0x55acbfa2fd7d <unknown>\n#14 0x55acbfa108c3 <unknown>\n#15 0x55acbf9de6b3 <unknown>\n#16 0x55acbf9df68e <unknown>\n#17 0x55acbfc80a2b <unknown>\n#18 0x55acbfc849b1 <unknown>\n#19 0x55acbfc6d225 <unknown>\n#20 0x55acbfc85532 <unknown>\n#21 0x55acbfc5238f <unknown>\n#22 0x55acbfca4f28 <unknown>\n#23 0x55acbfca50f3 <unknown>\n#24 0x55acbfcb4e7c <unknown>\n#25 0x7fe01d894ac3 <unknown>\n\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 22, in main_page_open\n    driver.get(MainPageLinks.URL_MAIN_PAGE)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 353, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py\", line 344, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "driver",
        "value" : "<selenium.webdriver.chrome.webdriver.WebDriver (session=\"f82ff9a3a5d926057fa82c59f5b345f9\")>"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : true,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  }, {
    "name" : "driver",
    "time" : {
      "start" : 1728694576822,
      "stop" : 1728694577588,
      "duration" : 766
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "afterStages" : [ {
    "name" : "driver::0",
    "time" : {
      "start" : 1728694578254,
      "stop" : 1728694578316,
      "duration" : 62
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "Test Header"
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "header_test"
  }, {
    "name" : "subSuite",
    "value" : "TestHeaderPage"
  }, {
    "name" : "host",
    "value" : "fv-az1245-423"
  }, {
    "name" : "thread",
    "value" : "2340-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.header_test"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 2,
        "skipped" : 0,
        "passed" : 52,
        "unknown" : 0,
        "total" : 54
      },
      "items" : [ {
        "uid" : "f1fd7b4faa3d1808",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/149//#testresult/f1fd7b4faa3d1808",
        "status" : "broken",
        "statusDetails" : "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x55833d7f102a <unknown>\n#1 0x55833d4d75e0 <unknown>\n#2 0x55833d4cf7eb <unknown>\n#3 0x55833d4bfa09 <unknown>\n#4 0x55833d4c1744 <unknown>\n#5 0x55833d4bfcbd <unknown>\n#6 0x55833d4bf52f <unknown>\n#7 0x55833d4bf422 <unknown>\n#8 0x55833d4bd493 <unknown>\n#9 0x55833d4bdada <unknown>\n#10 0x55833d4d9f41 <unknown>\n#11 0x55833d56b8c5 <unknown>\n#12 0x55833d54bb22 <unknown>\n#13 0x55833d56ad7d <unknown>\n#14 0x55833d54b8c3 <unknown>\n#15 0x55833d5196b3 <unknown>\n#16 0x55833d51a68e <unknown>\n#17 0x55833d7bba2b <unknown>\n#18 0x55833d7bf9b1 <unknown>\n#19 0x55833d7a8225 <unknown>\n#20 0x55833d7c0532 <unknown>\n#21 0x55833d78d38f <unknown>\n#22 0x55833d7dff28 <unknown>\n#23 0x55833d7e00f3 <unknown>\n#24 0x55833d7efe7c <unknown>\n#25 0x7f2808494ac3 <unknown>",
        "time" : {
          "start" : 1728608189266,
          "stop" : 1728608189266,
          "duration" : 0
        }
      }, {
        "uid" : "2b5c5bb2aab95b90",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/322//#testresult/2b5c5bb2aab95b90",
        "status" : "passed",
        "time" : {
          "start" : 1728556987982,
          "stop" : 1728556988176,
          "duration" : 194
        }
      }, {
        "uid" : "7bd2a469026c15fb",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/321//#testresult/7bd2a469026c15fb",
        "status" : "passed",
        "time" : {
          "start" : 1728553902542,
          "stop" : 1728553902719,
          "duration" : 177
        }
      }, {
        "uid" : "730493241e61791f",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/148//#testresult/730493241e61791f",
        "status" : "passed",
        "time" : {
          "start" : 1728522011224,
          "stop" : 1728522011433,
          "duration" : 209
        }
      }, {
        "uid" : "456b6d03102fc271",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/320//#testresult/456b6d03102fc271",
        "status" : "passed",
        "time" : {
          "start" : 1728470455025,
          "stop" : 1728470455235,
          "duration" : 210
        }
      }, {
        "uid" : "73c127e6179b5093",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/319//#testresult/73c127e6179b5093",
        "status" : "passed",
        "time" : {
          "start" : 1728467869515,
          "stop" : 1728467869709,
          "duration" : 194
        }
      }, {
        "uid" : "fff0782a438a17c0",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/147//#testresult/fff0782a438a17c0",
        "status" : "passed",
        "time" : {
          "start" : 1728435728890,
          "stop" : 1728435729085,
          "duration" : 195
        }
      }, {
        "uid" : "3657fc8dd12d309e",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/318//#testresult/3657fc8dd12d309e",
        "status" : "passed",
        "time" : {
          "start" : 1728377973810,
          "stop" : 1728377974005,
          "duration" : 195
        }
      }, {
        "uid" : "43414e251a62de4",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/317//#testresult/43414e251a62de4",
        "status" : "passed",
        "time" : {
          "start" : 1728375178231,
          "stop" : 1728375178436,
          "duration" : 205
        }
      }, {
        "uid" : "fb888cd4df044e40",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/316//#testresult/fb888cd4df044e40",
        "status" : "passed",
        "time" : {
          "start" : 1728375064301,
          "stop" : 1728375064480,
          "duration" : 179
        }
      }, {
        "uid" : "159a232751e97479",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/146//#testresult/159a232751e97479",
        "status" : "passed",
        "time" : {
          "start" : 1728349144133,
          "stop" : 1728349144338,
          "duration" : 205
        }
      }, {
        "uid" : "78f1a42265690459",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/315//#testresult/78f1a42265690459",
        "status" : "passed",
        "time" : {
          "start" : 1728292722958,
          "stop" : 1728292723150,
          "duration" : 192
        }
      }, {
        "uid" : "b8f6c8fde684ae91",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/314//#testresult/b8f6c8fde684ae91",
        "status" : "passed",
        "time" : {
          "start" : 1728289768763,
          "stop" : 1728289768982,
          "duration" : 219
        }
      }, {
        "uid" : "3d721169cff50c2a",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/145//#testresult/3d721169cff50c2a",
        "status" : "passed",
        "time" : {
          "start" : 1728263175377,
          "stop" : 1728263175580,
          "duration" : 203
        }
      }, {
        "uid" : "e3e939edbafc1fa1",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/313//#testresult/e3e939edbafc1fa1",
        "status" : "passed",
        "time" : {
          "start" : 1728226304635,
          "stop" : 1728226304806,
          "duration" : 171
        }
      }, {
        "uid" : "d485ddc709348423",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/312//#testresult/d485ddc709348423",
        "status" : "passed",
        "time" : {
          "start" : 1728221432183,
          "stop" : 1728221432393,
          "duration" : 210
        }
      }, {
        "uid" : "e6ebe17fb863cf7b",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/144//#testresult/e6ebe17fb863cf7b",
        "status" : "passed",
        "time" : {
          "start" : 1728176709254,
          "stop" : 1728176709446,
          "duration" : 192
        }
      }, {
        "uid" : "7bbc80bb3a40c846",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/311//#testresult/7bbc80bb3a40c846",
        "status" : "passed",
        "time" : {
          "start" : 1728117377280,
          "stop" : 1728117377452,
          "duration" : 172
        }
      }, {
        "uid" : "c00c21486326e706",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/310//#testresult/c00c21486326e706",
        "status" : "passed",
        "time" : {
          "start" : 1728108770546,
          "stop" : 1728108770722,
          "duration" : 176
        }
      }, {
        "uid" : "f4bae38f085986d7",
        "reportUrl" : "https://Brain-up.github.io/auto-tests-python/143//#testresult/f4bae38f085986d7",
        "status" : "passed",
        "time" : {
          "start" : 1728090025167,
          "stop" : 1728090025347,
          "duration" : 180
        }
      } ]
    },
    "tags" : [ ]
  },
  "source" : "16ba88760c75605f.json",
  "parameterValues" : [ ]
}